packages/twenty-server/src/engine/integrations/exception-handler/hooks/use-exception-handler.hook.ts

```diff
@@ -125,5 +130,24 @@ export const useExceptionHandler = <PluginContext extends GraphQLContext>(
         },
       };
     },
+    onValidate: ({ context, validateFn, params: { documentAST, schema } }) => {
+      const errors = validateFn(schema, documentAST);
+
+      if (Array.isArray(errors) && errors.length > 0) {
+        // const {workspace} = await options.createContextFactory?.create(context);
```

Comment:
@rostaklein You can do something like:
```typescript
context.workspace
```
As `user` and `workspace` should be injected inside the context already

Reply:
Unfortunately none of `user` or `workspace` are available at this point. Looks like the context function from the YogaDriver gets called only once the schema is valid.